week3/homework/src/index.js

```diff
@@ -1,3 +1,102 @@
-'use strict';
+"use strict";
+
+const express = require("../node_modules/express");
+const uuidv4 = require("../node_modules/uuidv4");
+
+const app = express();
+
+app.use(express.json());
+
+const { writeFile, readFile } = require("./todo");
+
+app.get("/todos", function (request, response) {
+    readFile()
+        .then(function (data) {
+            response.json(data);
+            response.end();
+        })
+})
+
+
+app.post("/todos", function (request, response) {
+    readFile()
+        .then(function (todos) {
+            const newID = uuidv4();
+            const aTodo = request.body.todo;
+            aTodo.id = newID;
+            todos.push(aTodo);
+            writeFile(todos)
+                .then(function () {
+                    response.send("Wrote a todo");
+                    response.end();
+            })
+    })
+})
+
+
+app.delete("/todos/:id", function (request, response) {
+    const wantedID = request.params.id;
+    readFile()
+        .then(function (todos) {
+            let filteredTodos = todos.filter((todo) => todo.id !== wantedID);
+            return writeFile(filteredTodos)
+                .then(function () {
+                    request.send(`Todo with ID: ${wantedID} has been successfully removed`);
+                    request.end();
+            })
+    })
+})
+
+
+app.delete("/todos", function (request, response) {
+    readFile()
```

Comment:
Why are you reading the file and not doing anything with the answer before overwriting it?